### PR TITLE
add wolfSSL_Init to csr example, required for fips ecc

### PR DESCRIPTION
**Pull request by jpbland1**
ZD 16126

### CODE REVIEW

File: examples/csr/csr.c [hunk 1]
```diff
@@ -166,6 +166,13 @@ int TPM2_CSR_ExampleArgs(void* userCtx, int argc, char *argv[])
         argc--;
     }
 
+    /* init wolfssl, required for fips */
+    rc = wolfSSL_Init();
```
Comment (dgarske) [inline examples/csr/csr.c:170]: This is great, but a couple things.
1) In WOLFCRYPT_ONLY this needs to be `wolfCrypt_Init()`.
2) Use the wolfSSL values `WOLFSSL_SUCCESS`.
3) If you are going to call init you also need to call `_Cleanup()`...

**Review by dgarske — CHANGES_REQUESTED** [review]

File: examples/csr/csr.c [hunk 2]
```diff
@@ -166,9 +166,30 @@ int TPM2_CSR_ExampleArgs(void* userCtx, int argc, char *argv[])
         argc--;
     }
 
+    /* init wolfssl, required for fips */
+#ifdef WOLFCRYPT_ONLY
```
Comment (dgarske) [inline examples/csr/csr.c:170]: This logic should be in `wolfTPM2_Init` and `wolfTPM2_Cleanup`? I am pretty sure these init/cleanups are already called. Perhaps the issue is I only call the wolfCrypt_Init version?

**Review by dgarske — CHANGES_REQUESTED** [review]

File: examples/csr/csr.c (hunk 2 above)
Comment (dgarske) [reply examples/csr/csr.c:170]: Please swap out the logic you added in csr.c, but put into tpm2.c and add the `#ifdef WOLFCRYPT_ONLY` logic. Call wolfSSL_Init by default unless WOLFCRYPT_ONLY is defined.

Comment (jpbland1) [reply examples/csr/csr.c:170]: https://github.com/wolfSSL/wolfssl/blob/master/src/ssl.c#L6421
I think this is why, I'm going to call `wolfSSL_Init` when not `WOLFCRYPT_ONLY` and call `wolfCrypt_Init` and `wc_SetSeed_Cb` when it is

**Comment by jpbland1** [comment]
So you don't need to call wolfSSL_Init, I just needed to add the `wc_SetSeed_Cb` call if `WC_RNG_SEED_CB` is defined

**Review by dgarske — APPROVED** [review]

**Pull request merged**